scheduler/src/cook/components.clj

```diff
@@ -338,9 +337,12 @@
     (mount/start-with-args (cook.config/read-config config-file-path))
     (pool/guard-invalid-default-pool (d/db datomic/conn))
     (metrics-jvm/instrument-jvm)
-    (let [server (scheduler-server config)]
+    (let [server (scheduler-server config)
+          framework-id (get-in server [:framework-id])]
       (intern 'user 'main-graph server)
-      (log/info "Started Cook, stored variable in user/main-graph"))
+      (log/info "Started Cook, stored variable in user/main-graph")
+      (log/info "Framework-id " )
```

Comment:
```suggestion
      (log/info "Framework-id " framework-id)
```

Reply:
I wanted to remove the field from the job structure, but that change needs to be reverted as it turned out to be more invasive than expected. I also reverted the sandbox changes. No reason to change them now and again when we put it into the task structure.

Reply:
:) Like that change.